### PR TITLE
Update cost to const

### PR DESCRIPTION
**Pull request by hollyglot**

### CODE REVIEW

File: exercises/01_block-scoping.test.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ test('can modify the value of a `let` variable', () => {
 test('cannot modify the value of a `const` variable', () => {
   var releaseName = 'ES6'
   // This doesn't even compile, so we can't actually test this...
-  // once you've changed the `var` above to `cost`, comment out the line below
+  // once you've changed the `var` above to `const`, comment out the line below
```
Comment (kentcdodds) [inline exercises/01_block-scoping.test.js:18]: Wonderful! Thanks for letting me know! If you'd like, could you make this change in `templates/`? The `exercises` directory is generated :)

**Review by kentcdodds — CHANGES_REQUESTED** [review]

**Comment by gudmundurmar** [comment]
Ah, was just about to open this too. Fixing this ASAP would be drowning in PRs :)
Just a quick question though @kentcdodds. How are you generating the exercises through templates and why?

**Comment by kentcdodds** [comment]
I wrote a tool called [`split-guide`](http://git.io/split-guide) to generate it because it makes it easier to keep the exercieses and exercieses-final directories in sync :) It's actually pretty brilliant :)

**Comment by kentcdodds** [comment]
I'll go ahead and make this change myself. Thanks!

**Pull request closed without merging**